Tools/astyle/check_code_style.sh

```diff
@@ -4,8 +4,8 @@ FILE=$1
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 if [ -f "$FILE" ]; then
-	${DIR}/fix_code_style.sh --dry-run $FILE | grep --quiet Formatted
-	if [[ $? -eq 0 ]]; then
+	CHECK_FAILED=$(${DIR}/fix_code_style.sh --dry-run --formatted $FILE)
+	if [[ $CHECK_FAILED ]]; then
```

Comment:
Can you check for an empty string like this: `if [ -n "$CHECK_FAILED" ]; then`?

Reply:
Makes sense, I changed it.
For anyone else interested here's the explanation for the -n parameter in the man pages of `test`: https://linux.die.net/man/1/test